src/components/TodoList/TodoList.tsx

```diff
@@ -1,22 +1,84 @@
-import React from 'react';
+import React, { useRef } from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
+  todos: Todo[];
+  onTodo: (u: Todo) => void;
+  onCurrentIcon: (v: HTMLElement) => void;
```

Comment:
The `onCurrentIcon` prop expects an `HTMLElement`, but it might receive `null` if the ref is not set. Consider updating the prop type to `(v: HTMLElement | null) => void` to handle this scenario safely.